### ISSUE TITLE
HPC Carpentry - a way forward

### ISSUE THREAD

**Issue by ocaisa** [carpentrycon/carpentryconhome-proposals#33]
### Session details

* Session type: Breakout discussion

* Keywords: HPC, HPCCarpentry, cluster, incubator

* Permission to record this session: Yes

### Abstract

At CarpentryCon 2018, the community expressed the will and desire to create a 
Carpentry inspired curriculum for teaching new users of *High Performance Computing* 
systems how to use such resources effectively. Subsequently the [HPC Carpentry](https://github.com/hpc-carpentry) GitHub 
team was created and relevant existing material was repurposed and integrated into a single home. By mid 2020, development in the
hpc-carpentry repos has stalled, with only a subset of the original goals met.

In Europe, there are at least two initiatives that have the potential to provide the
human resources to refine and extend the teaching material, but with clear goals of
their own derived from their own context. We would like to involve the community in a
discussion on how to encourage and integrate such initiatives to help sustain the 
development of the existing material, and extend it to new topics.

For this, we propose a breakout session composed of
* a review of activity to date
* series of lightning talks by different stake holders
* an open discussion on how to move forward.

### Personal details

* Name or pseudoname of the session lead: See below.

* Co-leads' names (we recommend involving 2 helpers/co-leads): Alan O'Cais, Peter Steinbach

* Email or other ways to contact the session leads/co-leads: a.ocais@fz-juelich.de, p.steinbach@hzdr.de

* Country of residence and/or compatible Time Zones (provide options): Spain, CEST

* Would you like to present this multiple times, in other time zones: WRITE HERE 

* Would you like to volunteer to be listed as a wrangler/host for your time zone: WRITE HERE  

### Is there any help you would like to invite from the community? Please provide below in bullet points.

Anyone is welcome to chime in and make a contribution. We would really love to have lightning talks from
(potential) stakeholders, e.g., those involved in a project or initiative whose goals
may (or could) align with those of HPC Carpentry.

* Other comment:

**Comment by ChristinaLK** [comment]
thanks for submitting @ocaisa ! We're hoping to direct the community here for involvement and feedback next week - can you add another comment after mine that reiterates your ask for help + any other feedback you'd like from the community?

**Comment by ocaisa** [comment]
Thanks @ChristinaLK 

Anyone is welcome to chime in and make a contribution. We are particularly open to have lightning talks from:
*  (potential) stakeholders, e.g., those involved in a project or initiative whose goals may (or could) align with those of HPC Carpentry
* people who have delivered the material from [HPC Carpentry](https://github.com/hpc-carpentry), and any challenges that they encountered

**Comment by Sabryr** [comment]
Very nice to see this effort. Main reason for our fork to diverge so much was the delay in getting pull requests accepted/refused.  More than happy to contribute to the session in any way if you need it.  Not sure how the resonances are collected, reply to this thread or should one email

**Comment by ocaisa** [comment]
@Sabryr I think this experience would be a really valuable contribution to the discussion. Either email or the thread are both fine for me, I'll connect with @psteinb once we've had a round of feedback (and if the session gets accepted) to decide the details of how we organise ourselves.

**Comment by megan-guidry** [comment]
I'm keen to get involved with this! Happy to do a lightning talk on what we are hoping for in the AU/NZ region re: HPC Carpentry.  Time zone is tricky though, would have to be 9 or 10am CEST (for a 7 or 8pm start in NZ)... I could facilitate a AU/NZ/ Pacific friendly time.

**Comment by methylnick** [comment]
I would be keen to be involved too, from Downunder Australia.

**Comment by orchid00** [comment]
I'm interested to join, also AEST Downunder Australia. Thanks for pointing out @megan-guidry

**Comment by levlafayette** [comment]
Adding my name to the collection of people from AU/NZ;  AEST also. 

Hat-tip to @megan-guidry

**Comment by AnnBackhaus** [comment]
Adding my name to the Aus/NZ collection, AWST. 
Thanks @megan-guidry !

**Comment by prosoitos** [comment]
I would be interested to attend this conversation from Vancouver, BC, Canada.

**Comment by anandagk** [comment]
I would like to be involved and adding my name to AU/NZ - for Central time zone.
Thank you!

**Comment by psteinb** [comment]
@tkphd expressed interest from the US east coast. 

@ChristinaLK should we add the respective label then? Along this line of thought, maybe 2 breakouts for this might make sense. What do you think?

**Comment by tkphd** [comment]
Absolutely @psteinb, happy to chime in from `Americas/New_York` (UTC-5:00). 
I've contributed a *little* to the material. Embarrassingly, I have not yet yet run the HPC Carpentry curriculum, though I did run a very heavily modified GPU-centric derivative work.

**Comment by serahkiburu** [comment]
Hi @ocaisa,  the organising committee reviewed this session proposal and had this to say:

>The session leads have led an effort to develop HPC curriculum in the past and any help we can get to revive their work on the HPC curriculum is highly welcome. In that regard, we are very grateful to see this proposal. If possible, we recommend that authors add some details on how to continue engaging with their participants after this session. 

We are really excited for your session, and more details i.e. around scheduling will be shared with you in the coming days. Let me know if I can answer any questions or clarify anything for the time being. Thanks!

**Comment by reid-a** [comment]
I'd be delighted to get back into this. I am, like @tkphd, also on the US east coast -- I'm fine with a second breakout if there's enough interest, of course.

A long-standing "to do" for me is to exercise the customization/localization machinery by standing up a test fork for my own cluster, and provide feedback on how that goes.

**Comment by psteinb** [comment]
@serahrono and @ChristinaLK the proposed date on July 21 won't work out from our point of view. We have interested parties from Australia, Europe and the US in this thread. We hence propose to go for:
July 20, 7am UTC for Europe+Australia/NZ timezones
July 20, 3pm UTC for Europe+US timezones
I already indicated this in the google sheet, but wanted to communicate that here to underline that we intend to conduct the session twice.

**Comment by psteinb** [comment]
I updated the dates in my previous comment. Had a typo there that I just noticed! Sorry. We are aiming at July 20, 2020. Gdoc edited as well.

**Comment by Sabryr** [comment]
Do we have a Gdoc ?

**Comment by psteinb** [comment]
Yes, this is currently used within carpentrycon to arrange the dates. As this is a moving target and I am not the author, I didn't feel like publishing a link to it.

**Comment by reid-a** [comment]
So I have been busy for the past week, and am just now coming back to this -- it appears there are two CarpentryCon HPC breakouts scheduled for July 20, 2020, at 07h00 UTC and 15h00 UTC, based on the CarpentryCon [schedule](https://2020.carpentrycon.org/schedule/) with a good agenda and a link to an Etherpad.

Thanks for setting that up, looking forward to the connection details.

**Comment by ocaisa** [comment]
@psteinb @reid-a @Sabryr @tkphd @anandagk @prosoitos @AnnBackhaus @levlafayette @orchid00 @methylnick @megan-guidry 

Hi to all who've shown interest, as an update from our side, we've added the proposed agenda to [the session etherpad](https://pad.carpentries.org/cchome-hpc-carpentry). Please go sign up and check things out there.

Note that there is a section in there called **"Please help us!"**, we want to discuss 4 themes in the different breakout rooms and we  are asking people to add topics for discussion under each theme. We know these lists might get long, so we will filter a bit to remove duplication on Friday and take a poll during the session to know which points are most pressing. Please, please add some topics in there or we will have an echo chamber!
